### ISSUE TITLE
Unknown outgoing transaction form my chia wallet

### ISSUE THREAD

**Issue by BRNGZ** [Chia-Network/chia-blockchain#8010]
When I check my wallet history, I saw a outgoing transaction which does not belongs to me. How can it happen?
Anyone having the same problem?

![Screenshot 2021-08-10 195947](https://user-images.githubusercontent.com/88734316/128902472-310470ba-b815-4869-8005-f8d06ce4484d.png)

**Comment by loppefaaret** [comment]
That seems to be a transaction, made on that machine ?
If you yourself have not created it, any chance that you got any kind of remote access software running, that potentially has been compromised ?
Do any one else have physical access to the machine ?
The address also sent out a transaction earlier today ?
https://xchscan.com/address/xch1fvfzg0f6jdr9dmwz60kcn8jk6yngg0vr6takwq8fees53udrdm0qaxtt2s

**Comment by BRNGZ** [comment]
No one actually have physical access to my machine. 
The second transaction was made by me to in order to keep the left coins safe :( 
I already read lots of comments about "suddenly disappeared coins" from people's wallets. Most of them seemed to be because of some scripts they were using even like swar.
Even I only used madmax, I think, I got the similar problem.
There seems some solutions but "very hard", reinst OS, delete all keys which may cause to replot all current plots"
It seems a very big "security problem" of chia blockchain".  Normally, a transaction should need additional security steps.
There are so many software in "github" and people can not know which is really safe and which is not. A script including 1-2 lines sending all your information to someone seems very easy to steal our info if you are not familiar with scripts or trust because many people are using them.
